src/components/common/Extensions/ThemeSwitcher/ThemeSwitcher.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import clsx from 'clsx';
+import Icon from '~shared/Icon';
+import styles from './ThemeSwitcher.module.css';
+import { formatMessage } from '~utils/yup/tests/helpers';
+
+const ThemeSwitcher = () => {
+  const [isLightMode, setIsLightkMode] = useState<boolean>(true);
```

Comment:
type explicitly is not necessary, just useState(true)

Reply:
Type removed